src/utils/validate.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * Checks if Valid default coordinates
+ * @param {unknown} possibleCoordinates
```

Comment:
Can we give to this file a more precise name? `validate-schema.js`, `validate-type.js`